warehouse/models/mart/transit_database/dim_transit_data_quality_issues.sql

```diff
@@ -0,0 +1,41 @@
+{{ config(materialized='table') }}
+
+WITH int_transit_database__transit_data_quality_issues AS (
+    SELECT * FROM {{ ref('int_transit_database__transit_data_quality_issues') }}
+),
+
+dim_transit_data_quality_issues AS (
+    SELECT
+        key,
+        source_record_id,
+        description,
+        issue_type,
+        gtfs_dataset,
```

Comment:
This is where I think we still may need a bridge table, because ideally this would be a `gtfs_dataset_key` value, rather than a `source_record_id` value (I think if you just unnest and keep the outcome, you're going to have the `source_record_id` for the associated GTFS dataset, which can map to multiple actual records during this time period if updates were being made to that foreign record.)

The purpose of the bridge table would be to provide a 1:1 **versioned** relationship for this issue to all associated GTFS dataset records. Even though this is current-only, because it has the fake `_valid` dates, it will join with a lot of versions of the associated records if you just join on `source_record_id`